examples/collector-exporter-node/start.js

```diff
@@ -2,12 +2,14 @@
 
 const opentelemetry = require('@opentelemetry/api');
 const { BasicTracerProvider, ConsoleSpanExporter, SimpleSpanProcessor } = require('@opentelemetry/tracing');
-const { CollectorTraceExporter } = require('@opentelemetry/exporter-collector');
+const { CollectorTraceExporter, CollectorProtocolNode } = require('@opentelemetry/exporter-collector');
 
-const address = '127.0.0.1:55680';
 const exporter = new CollectorTraceExporter({
   serviceName: 'basic-service',
-  url: address,
+  // headers: {
```

Comment:
Why this is commented out? Custom headers and metadata feature released in `0.9.0`.

Reply:
this is an example to be able to test it or not